src/api.js

```diff
@@ -1,5 +1,7 @@
 import axios from "axios";
 
+const username = 'happyamy2016';
```

Comment:
instead of hardcoding this - could we have a button that logs you in as this user?
That way, could build in some functionality that stops you posting if not logged in.
Can also build this out to be a user select/login at a later date